ddl/column_test.go

```diff
@@ -740,24 +754,33 @@ func (s *testColumnSuite) TestAddColumn(c *C) {
 
 	newColName := "c4"
 	defaultColValue := int64(4)
+
+	var mu sync.Mutex
+	var hookErr error
 	checkOK := false
 
 	tc := &testDDLCallback{}
-	var checkErr error
 	tc.onJobUpdated = func(job *model.Job) {
+		mu.Lock()
```

Comment:
We need to access `hookError` inside and outside hook function. hook function is run in different goroutine.